go.mod

```diff
@@ -74,6 +74,7 @@ require (
 	github.com/prometheus/client_golang v1.5.1
 	github.com/prometheus/client_model v0.2.0
 	github.com/prometheus/common v0.9.1
+	github.com/retailnext/hllpp v1.0.0
```

Comment:
Sanity-checking: this dependency was used in 1.x for this command?

Reply:
Yes it was used in the 1.x version of this command